assets/l10n/app_en.arb

```diff
@@ -23,6 +23,10 @@
   "@switchAccountButton": {
     "description": "Label for main-menu button leading to the choose-account page."
   },
+  "aboutZulipMenuItem": "About Zulip",
+  "@aboutZulipMenuItem": {
+    "description": "Label for the 'About Zulip' option in the main menu."
+  },
```

Comment:
We already have one "About Zulip" button. Does its label have an entry in this file? If so, probably good to reuse that instead of adding a new entry. If its description is specific to that other place where the button appears, please adjust it because it won't be correct after reusing it in this new place.

Reply:
@chrisbobbe `aboutPageTitle` exists already
```json
"aboutPageTitle": "About Zulip",
  "@aboutPageTitle": {
    "description": "Title for About Zulip page."
  },
```
May I adjust the description to "Title and main menu option label for About Zulip page"

Reply:
Since its description is not specific to that other place where the button appears—it's about the page itself, not the button that leads to it—it's still correct after adding a button in the new place. The description doesn't need to change.